x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/analysis/analyzer/Analyzer.java

```diff
@@ -124,7 +124,6 @@ public Analyzer(Configuration configuration, FunctionRegistry functionRegistry,
                 //new ImplicitCasting()
                 );
         Batch finish = new Batch("Finish Analysis",
-                new PruneSubqueryAliases(),
```

Comment:
Shouldn't we remove this inner class? is it used anywhere else?

Reply:
Still required. it was moved to the Optimizer.